docs/Report.md

```diff
@@ -111,34 +88,31 @@ Then, confirmation of success is sent back until the razorpage redirects to itse
 <!-- Describe briefly the illustration, i.e., how you application is built, tested, released, and deployed. -->
 ### GitHub workflows
 
-To ensure the flow of the project, we use a tool developed by GitHub known as. GitHub Action, otherwise known as workflow. This will also entail when the workflows are activated and used.
+To ensure the flow of the project, we use a tool developed by GitHub known as. GitHub Action, otherwise known as a workflow. This will also entail when the workflows are activated and used.
 
 #### Build and Test
 
-The build and test workflow can be found in . The activity diagram shows how GitHub ensures what is merged into main. This workflow is run on a pull request every time a commit is made to the branch in the pull request. This is to ensure that main will still work by building the project with dotnet and tests made for the project.
- Because it runs the tests as well, it ensures that any incoming changes do not affect the functionality. If anything fails, it will stop and prevent the branch from merging into main.
-<figure>
-   <center> <img src="Images/BuildAndTest.png"
-         alt="BuildAndTest Activity diagram">
-    <figcaption>Fig.XX Build and test activity diagram</figcaption></center>
-</figure>
+The build and test workflow can be found [here](#BuildTest). The activity diagram shows how GitHub ensures what is merged into main does not damage it. 
+This workflow is run on a pull request every time a commit is made to the branch belonging to the pull request. 
+This ensures that main will stay functional by building and testing the project with dotnet and our test suite. If anything fails, it will stop and prevent the branch from merging into main.
```

Comment:
Rewrote to increase clarity and brevity.

Reply:
The reference to the diagram does not work on a PDF

Reply:
That is unfortunate, My pandoc does currently not work so can't test